styles.css

```diff
@@ -23,17 +23,22 @@ margin-top: 10%;
   color: #E9E9F3;
   font-weight: bold;
   height: 5%;
+  margin: 0 10px;
 }
 
 h1 {
```

Comment:
We have multiple h1s. Do some of these need to have class names to get more specific